tests/forte/image_annotation_test.py

```diff
@@ -56,3 +56,41 @@ def test_image_annotation(self):
         self.assertEqual(
             new_pack.audio_annotations, self.datapack.audio_annotations
         )
+
+    def testBox(self):
+
+        b1 = Box(self.datapack, [2, 2], [7, 7], 0)
+        self.assertEqual(b1.corners, [(2, 2), (2, 7), (7, 2), (7, 7)])
+        self.assertEqual(b1.area, 25)
+        self.assertEqual(b1.center, (4, 4))
+
+        b2 = Box.init_from_center_n_shape(self.datapack, 4, 4, 5, 5)
+        self.assertEqual(b2.corners, [(2, 2), (2, 7), (7, 2), (7, 7)])
+        self.assertEqual(b2.area, 25)
+        self.assertEqual(b2.center, (4, 4))
+
+        def wrong_box():
+            # negative x coordinate
+            Box(self.datapack, [-2, 2], [7, 7], 0)
+
+        self.assertRaises(ValueError, wrong_box)
+
+        def wrong_box():
+            # negative y coordinate
+            Box(self.datapack, [2, -2], [7, 7], 0)
+
+        self.assertRaises(ValueError, wrong_box)
+
+        def wrong_box():
+            # negative width
+            Box(self.datapack, [3, 2], [2, 7], 0)
+
+        self.assertRaises(ValueError, wrong_box)
+
+        def wrong_box():
+            # negative height
+            Box(self.datapack, [2, 2], [2, 1], 0)
+
+        self.assertRaises(ValueError, wrong_box)
+
+        b3 = BoundingBox.init_from_center_n_shape(self.datapack, 4, 4, 5, 5)
```

Comment:
I guess it's the same as `Box.init_from_center_n_shape`

Reply:
It shows the child class can init from center and shape as well, and it keeps the child class type.